2024/04.md

```diff
@@ -129,6 +129,7 @@ When applicable, use these emoji as a prefix to the agenda item topic.
     |   | 1     | 30m     | `eval`/`new Function` changes for Trusted Types as Normative PR or Stage 3 ([PR](https://github.com/tc39/ecma262/pull/3294), [slides](https://docs.google.com/presentation/d/14AjvbW2-aNvlirB-7pPhIAeM7oWKtjqONRcyp9ID7gg/edit?usp=sharing)) | Nicolò Ribaudo |
     |   | 1     | 30m     | [ESM Source Phase](https://github.com/tc39/proposal-esm-phase-imports) status update | Guy Bedford |
     |   | 1     | 30m     | [Module sync assert](https://github.com/tc39/proposal-module-sync-assert) for stage 2 | Jack Works |
+    |   | 1     | 30m     | [Intl.MessageFormat](https://github.com/tc39/proposal-intl-messageformat) status update, maybe for stage 0? | Eemeli Aro |
```

Comment:
stage 0? is that correct?

Reply:
Yes. I need to step down from championing this.